2-mandatory/2-retrying-random-numbers.js

```diff
@@ -5,13 +5,18 @@
 
 // This function shouldn't be changed
 function generateRandomNumber() {
-    console.log("Generating number...");
+    console.log("Generating number...")
     return Math.round(Math.random() * 100);
-}
-
-function getRandomNumberGreaterThan50() {
-    // TODO - implement using a do-while loop
-}
+ }
+ 
+ function getRandomNumberGreaterThan50() {
+    let numbersInArray = []
+    do{
+        numbersInArray = generateRandomNumber()
+    }while(numbersInArray<51)
```

Comment:
I'm not sure why you used an array here, it does seem to work but maybe you can teach me something after this review 😂 

I'd have gone for an integer / Number.